.github/workflows/ci.yml

```diff
@@ -193,7 +197,7 @@ jobs:
       fail-fast: false
       matrix:
         rust: [stable]
-        version: ["1.8", "1.10", "1.12", "1.13"]
+        version: ["1.8", "1.10", "1.12", "1.14"]
```

Comment:
Yea, probably the right call to just throw away 1.13 from the CI and replace it with 1.14 where possible.